Various EN stringtable fixes/cleaning/formatting

Ouaz opened the pull request.

Ouaz committed:
Insert omitted macros "building on Outposts"

Replace sentence "This building can be built on an [[encyclopedia OUTPOSTS_TITLE]]" by the macro [[BUILDING_AVAILABLE_ON_OUTPOSTS]].

Ouaz committed:
Formatting and typo fixes

- Add double lines before sections, remove unneccessary double lines
- Typos: verious > various (SP_REPLICON_DESC) ; automaitcally > automatically (DESIGN_WINDOW_ARTICLE_TEXT)

Ouaz committed:
Insert various pedia links

MatGB reviewed (commented):
Apart from C Camps, it looks good to me.

Ouaz committed:
Remove incorrect sentence in BLD_CONC_CAMP_DESC

- Remove "This structure can be built at an Outpost", as Concentration Camps require a Population of at least 3.

Ouaz committed:
Remove double spaces after periods

Discussed in #1807
